src/collectors/monit/monit.py

```diff
@@ -106,7 +106,7 @@ def collect(self):
                                 metrics["%s.memory_total.%s_usage" % (
                                     name, unit)] = (
                                     diamond.convertor.binary.convert(
-                                        value=mem,
+                                        value=mem_total,
```

Comment:
Yes it was a subtle bug I discovered. Probably should've been broken up into a separate commit